api/controllers/main.js

```diff
@@ -11,34 +11,62 @@ function loadAssetManifest() {
 
 let webpackAssets = loadAssetManifest();
 
+function defaultContext(req) {
+  if (process.env.NODE_ENV === 'development') {
+    // reload the webpack assets during development so we don't have to
+    // restart the server for front end changes
+    webpackAssets = loadAssetManifest();
+  }
+
+  const siteDisplayEnv = config.app.app_env !== 'production' ? config.app.app_env : null;
+
+  const messages = {
+    errors: req.flash('error'),
```

Comment:
Decided to structure this way so that we could more easily add other message types (like `info`) in the future, but keep them all contained in the `messages` object.